app/views/collections/ugc.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+module Collections
+  class Ugc < ApplicationView
+    include BrowsableView
+    include SearchableView
+
+    def js_vars
+      [
+        {
+          name: 'pageName', value: 'e7a_1418'
+        }
+      ]
+    end
+
+    def page_title
+      'Europeana - First World War'
+    end
+
+    def content
```

Comment:
Fair enough. I thought these were used for breadcrumbs, but if not, then remaining in `#content` makes sense.